pfb-analysis/scripts/setup_database.sh

```diff
@@ -7,15 +7,32 @@ NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
 # Set defaults for overridable configuration params
-PFB_WORK_MEM="${PFB_WORK_MEM:-2048MB}"
 PFB_CHECKPOINT_COMPLETION="${PFB_CHECKPOINT_COMPLETION:-0.8}"
-PFB_MAX_WAL_SIZE="${PFB_MAX_WAL_SIZE:-2GB}"
+# Disable to improve performance, if disabled, data loss occurs on server crash (ok for analysis)
+PFB_FSYNC="${PFB_FSYNC:-off}"
```

Comment:
Adding manual VACUUMs made no difference in run time. I'm going to leave them in since they don't hurt to have if the analysis algorithms change in the future.